fossology/fossology
# feat(copyright): add new keywords for ECC and keyword agent

## Description

Add new keywords to ECC and keyword agent.

## How to test

* Upload a package with new keywords and check if the agent is identifying them.

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2370"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
